### PR TITLE
Create Count Mase Path

### LINKED ISSUE #260: Count mase path in the chessboard
This program is the solution to a very popular question-mase path problem.
the language used *. 
 -
- [x] Java

### PR DESCRIPTION
**Pull request by Coding-Point**
Count mase path problem with java.

# Title/Heading < Edit this part>

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. < Edit this part>

Fixes #260 

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@Coding-Point do the following things:
- Raise an issue which this PR solves (Fill that template carefully)
- Wait for it to be assigned to you
- Fill this PR template accordingly
- Provide the file an extension

**Comment by kaustubhgupta** [comment]
@Coding-Point :
> * Fill this PR template accordingly
> * Provide the file an extension
> * Update java folder readme

File: countmasepath.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			for(int j=0;j<strg.length;j++) {
+				if(strg[i][j]/10==0) {
+				System.out.print(strg[i][j]+"   ");
+				}else {
+					System.out.print(strg[i][j]+"  ");
+					
+				}
+			}
+			System.out.println();
+		}
```
Comment (YashSaxena9) [inline countmasepath.java:21]: Can you specify the need of this nested loop, they seem to be useless.

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]

File: countmasepath.java (hunk 1 above)
Comment (kaustubhgupta) [reply countmasepath.java:21]: Also, make an entry in Java folder readme

**Comment by kaustubhgupta** [comment]
Due to inactivity, this PR has been closed

**Pull request closed without merging**